### ISSUE TITLE
Mapping using longranger

### ISSUE THREAD

**Issue by RNieuwenhuis** [bcgsc/arcs#71]
Hi,

I was wondering if mapping 10x reads using longranger (lariat) would yield better results in the scaffolding process. Because a barcode aware mapping should result in less mapping errors if I am not mistaken. I did see #2 but there is no conclusive answer in that thread. 

What is you opinion on this matter? 

Cheers,

**Comment by lcoombe** [comment]
Hi @RNieuwenhuis,

We haven't tested using lariat instead of `bwa mem` for the read alignment step of ARCS, so I can't give you a conclusive answer.

Using a barcode-aware aligner could lead to slightly better alignments, and thus possibly slightly better scaffolding, but my intuition is that this wouldn't be a big difference. That's because we only keep read alignments that are mapped in a proper pair and filter out mapping quality zero alignments, which should address most of the cases of mapping errors. My understanding is also that the barcode-aware aligners are not as efficient in terms of run-time, so the gains might not be worth the extra computational cost. 

If you do decide to try both `lariat` and `bwa mem` in ARCS, I'd be interested to see how your results compare!

Hope that helps,
Lauren

**Comment by RNieuwenhuis** [comment]
It has been a while but I saw this thread was mentioned in another issue. I have tried `bwa mem` , `lariat` (longranger align) and `emerald` (https://github.com/arshajii/ema) . I tested this on a cucumis species that was assembled using PacBio CLR reads subsequently polished using PacBio and 10X data. I concluded the effect of the used mapper is very small. Especially compared to the effect of different settings of arcs. So in my case @lcoombe 's intuition was right.
